### ISSUE TITLE
Not possible to overwritew code translation except of origin module 16.0 17.0

### ISSUE THREAD

**Issue by kamilzmija** [odoo/odoo#157815]
Impacted versions: 16, 17

Code translations are loaded only from the module in which the code is implemented, so there are 2 solutions - add translation in orginal module (stupid), inherit and overwrite code so "_lt()" will be executed in our module (even more stupid).

Code below clearly explains it
![obraz](https://github.com/odoo/odoo/assets/54367405/fd6869b3-4a2b-47fa-91ed-e9e83d0b6f7a)

**Comment by jappi00** [comment]
Same problem here. A solution to this would be great.

**Comment by dupski** [comment]
Yeah, this is a real limitation of the new translation system, makes it impossible to override e.g. Exception messages in custom modules.

**Comment by jappi00** [comment]
A good German translation in core and enterprise would solve the problem too. 😁

**Comment by AquaMCU** [comment]
Quick Fix: Import the PO file directly in settings. But we need to be able to have the old system, where a module can overwrite other translations directly.

**Comment by AquaMCU** [comment]
But then you run into this: 
https://github.com/odoo/odoo/issues/168098
